NASA-AMMOS/aerie-ui
# Feature: Upload external resources

`___REQUIRES_GATEWAY_PR___="112"`

Partially resolves: https://github.com/NASA-AMMOS/aerie-ui/issues/1424

To test:
1. Upload a model
2. Create a plan using that model
3. Open the "Activity & Resources" panel on that plan
4. Click the "Resources" tab in that panel
5. Click the "Upload Resources" icon button
6. Verify that a file upload input pops up below the button
7. Choose the test file `/e2e-test/data/external-dataset.json`
8. Click the "Upload" button
9. Verify that the new resources show up in the resources list below
10. Repeat steps 7-9 but with the `/e2e-test/data/external-dataset.csv` file
